README.md

```diff
@@ -109,10 +110,46 @@ class PostData extends DataTransferObject
 }
 ```
 
-By adding doc blocks to our properties, their values will be validated against the given type; 
-and a `TypeError` will be thrown if the value doesn't comply with the given type.
+Since PHP 7.4 introduced typed properties, you can use the new, built-in syntax. Their values will be validated against the given type and a `TypeError` will be thrown if the value does not comply with it.
```

Comment:
Re-reading the readme in its entirety, it strikes me that this should be already at the beginning of the readme where "Installation" is. _Usually_ a readme has something like "requirements" (this doens't) but I think the first thing mentioned is that it requires 7.4.

Following this, not sure if this sentence, explaining that 7.4 has typed properties, makes even sense because.. it's like news to use "now" but in fact you.

Maybe add the aforementioned to the beginning at the installation and then here something like:
> You can take full advantage of PHP 7.4 and its typed properties. Their values …

Reply:
Yeah I agree with you. I am not sure about throwing it under the installation section though. As a developer reading through, seeing built-in typing vs phpdoc typing back to back helps me see some of the feature differences.

Perhaps we need to revise the existing text to something along the lines of:

> When defining typed properties, you can take advantage of the built-in types supported by PHP. These properties will be validated against the given type and a `TypeError` will be thrown if the value does not comply with it.

And add this to the installation section:

> * **Note**: This package requires PHP 7.4 so it can take full advantage of type casting in PHP.

Reply:
> so it can take full advantage

I was about to say that's not correct but (7.4 is the minimum)… of course versions <2 do support < PHP 7.4 so 🤷‍♀️

Reply:
@mfn They are also tagged and should have a different README.

Reply:
Resolved by 70c516805f63e94297a3baab34f8771904be2ee5